pastastore/plotting.py

```diff
@@ -1318,4 +1410,5 @@ def add_labels(df, ax, adjust=False, **kwargs):
                     textcoords=textcoords,
                     xytext=xytext,
                     **{"path_effects": stroke},
+                    **kwargs,
```

Comment:
Won't this result in issues if kwargs are parsed to both adjust_text and annotate?

Reply:
Ugly I agree, but basically you cannot pass kwargs to annotate when adjust=True.

Reply:
I'll make a separate issue.